content/scalers/mysql.md

```diff
@@ -0,0 +1,101 @@
++++
+fragment = "content"
+weight = 100
+title = "MySQL"
+background = "light"
++++
+
+Scale applications based on MySQL query result.
+
+**Availability:** v1.1+ | **Maintainer:** Community
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `mysql` trigger that scales based on result of MySQL query.
+The trigger requires the following information:
+- `query` - a MySQL query that should return single numeric value that will be used for HPA
```

Comment:
We should add a line as the list is not rendered correctly, see https://deploy-preview-72--vigorous-franklin-8edd65.netlify.com/scalers/mysql/
```suggestion

- `query` - a MySQL query that should return single numeric value that will be used for HPA
```